compute_tools/src/sql/finalize_drop_subscriptions.sql

```diff
@@ -0,0 +1,21 @@
+DO $$
+BEGIN
+    IF NOT EXISTS(
+        SELECT 1
+        FROM pg_catalog.pg_tables
+        WHERE tablename = 'drop_subscriptions_done'
+        AND schemaname = 'neon_migration'
+    )
+    THEN
+        CREATE SCHEMA IF NOT EXISTS neon_migration;
+        ALTER SCHEMA neon_migration OWNER TO cloud_admin;
+        REVOKE ALL ON SCHEMA neon_migration FROM PUBLIC;
```

Comment:
This is probably another reason not to use the `neon_migration` schema. Migrations are executed async and we already have a code that handles that, we just cannot rely on it here. If we create some `neon` or `neon_internal` schema (if `neon` schema is created with neon extension) synchronously somewhere around creating a `cloud_admin`, then we can just rely on it here